openapi/paths/storefront/transactions@{id}@{token}@continue.yaml

```
@@ -0,0 +1,27 @@
+parameters:
+  - $ref: ../../components/parameters/resourceId.yaml
+  - $ref: ../../components/parameters/token.yaml
+post:
+  x-products:
+    - Storefront
+  tags:
+    - Transactions
+  summary: Continue processing transaction
+  operationId: StorefrontPostKycRequestContinue
+  x-sdk-operation-name: continueKyc
+  security:
+    - CustomerJWT: []
+  description: Continues processing transaction with specified ID, after all documents of KYC request with specified token have been uploaded.
```

Perhaps something like:
```suggestion
  description: Continues processing the by KYC interrupted transaction with specified ID, after all documents of KYC request with specified token have been uploaded.
```

---

I used related suggestion from Chris